lib/widgets/message_list.dart

```diff
@@ -369,8 +369,7 @@ class MessageListAppBarTitle extends StatelessWidget {
           Padding(
```

Comment:
```
theme: Replace colorMessageHeaderIconInteractive with title

Removed `colorMessageHeaderIconInteractive` from
`DesignVariables` and used `title` in place of it
as per the design guidelines.
discussion: https://github.com/zulip/zulip-flutter/pull/1213#discussion_r1908343739
```

We can remove most of the commit message as it repeats the title and the diff, similar to other commits.

Let's also add a link to the Figma design.